servo/servo.dockerfile

```diff
@@ -44,5 +40,8 @@ RUN sudo sed -i "s/\(Xvfb :.*\)$/\1 +extension RANDR +extension RENDER +extensio
 RUN git clone https://github.com/servo/servo
 WORKDIR servo
 
+# Configure Cloud9 to use Servo's source directory as workspace.
```

Comment:
Self-nit: @bnjbvr told me that `# Point Cloud9's workspace to Servo's source directory.` would be clearer.